server/src/config.ts

```diff
@@ -13,7 +11,7 @@ export class Config {
   public static TOKEN_EXPIRATION: string = "1d";
   public static TOKEN_ISSUER: string = "socobo";
   public static TOKEN_HEADER: string = "x-access-token";
-  public static DEFAULT_USER_IMAGE: string = "http://placehold.it/350x150";
+  public static DEFAULT_USER_IMAGE_ID: number = 1;
```

Comment:
I also think of this as profile pics but I think the default image is some kind of app data and the user profile pic is user data and I'm not sure if it is a good idea to mix them up.